jobdispatcher/processor.py

```diff
@@ -154,6 +148,12 @@ def run(self):
                 lock.wait()
 
         logger.debug("Total number of rebalances: ", counter_rebalances)
```

Comment:
logger.debug(f"Total number of rebalances: {counter_rebalances}")

Reply:
Molto strano che dia questo errore. E' un valore inizializzato a int, forse è un problema di loggin. Sostituisco con f-string